python/basix/numba_helpers.py

```diff
@@ -0,0 +1,56 @@
+try:
+    import numba
+except ImportError:
+    raise RuntimeError("You must have numba installed to use the numba helper functions.")
+
+
+@numba.njit
+def apply_dof_transformation(entity_transformations, cell_type, entity_dofs,
```

Comment:
Could you add a docstring? It's not so clear which arguments are in and which are out.